### ISSUE TITLE
Dockerfile for ubuntu desktop 

### ISSUE THREAD

**Issue by ghost** [games-on-whales/gow#176]
It would be nice to include an example Dockerfile of a full Ubuntu desktop (with dbus ) environment that allows easy installation of any software.

**Comment by ABeltramo** [comment]
I agree, there will definitely be an "empty" desktop image that will be available sooner rather than later. My only gripe with that is that it should be a starting point for anyone to make their own image with the software that they want instead of being the place where you install packages at runtime..  
The main issue with that approach is that, given the nature of Docker, everything you'll install will be wiped on a restart. If instead you make your own image that will be always ready no matter what.

**Comment by Asjnhbv** [comment]
2db016dc64f287fa541c97b922d20f493fedf403

**Comment by Asjnhbv** [comment]
2db016dc64f287fa541c97b922d20f493fedf403
